.tool-versions

```diff
@@ -0,0 +1,2 @@
+poetry 1.2.0b3
+python 3.9.11
```

Comment:
3.9.13?

Reply:
Nope, looks like version of python used in code is 3.9.11

It shouldn't matter, but if we're going to change it, we can change it as a separate change.